python/nav/django/settings.py

```diff
@@ -262,3 +264,40 @@
         from local_settings import *
     except ImportError:
         pass
+
+
+OIDC_AUTH = {
+    # Specify OpenID Connect endpoint. Configuration will be
+    # automatically done based on the discovery document found
+    # at <endpoint>/.well-known/openid-configuration
+    'OIDC_ENDPOINT': os.environ.get("OIDC_ENDPOINT"),
```

Comment:
If I am to understand the `drf-oidc-auth` docs, this actually fetches information about the public key to approve from this server?

Can we still support the potential requirement of allowing multiple signing keys (i.e. NAV's own or the third party authority's?)

Reply:
This is changed a bit how it works now, also environment variables is no longer used for this